store/in_memory_alert_def_store.go

```diff
@@ -5,21 +5,44 @@ import (
 )
 
 // inMemoryAlertStore - concrete implementation of the AlertDefStore interface
-type inMemoryAlertStore struct {
+type InMemoryAlertStore struct {
```

Comment:
the type should be private.
the constructor is public and exposes the interface which is public.